include/models.h

```diff
@@ -33,6 +33,8 @@ class Model {
             bool doEarlyStopping_ = false, int eosTokenId_ = -1, int padTokenId_ = -1, bool doSample_ = false,
             float temperature_ = 1.0, int topK_ = 50, float topP_ = 1.0);
 
+    void config(SearcherConfig &config_);
```

Comment:
When you have added the config api, but you didn't use it?

Reply:
This is necessary for C++ API since it's annoying that all parameters are passed when just the last one is changed. There are two ways now, I just choose another way in the example.